src/main/java/org/geysermc/discordbot/listeners/DumpHandler.java

```diff
@@ -258,7 +258,15 @@ public void onMessageReceived(@NotNull MessageReceivedEvent event) {
                 .addField("Autoconfigured remote?", (config.getBoolean("autoconfiguredRemote")) ? "Yes" : "No", true)
                 .setTimestamp(Instant.now())
                 .setColor(BotColors.SUCCESS.getColor())
-                .build()).queue();
+
+        if (dump.has("logsInfo") && !dump.isNull("logsInfo")) {
```

Comment:
`JSONObject#isNull()` returns false if the value doesn't exist, so you only need it

Reply:
Sounds right
```suggestion
        if (!dump.isNull("logsInfo")) {
```